pom.xml

```diff
@@ -557,19 +557,19 @@
             <plugin>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <source>${maven.compiler.source}</source>
+                    <source>1.8</source>
                     <doctitle>Eclipse Collections - ${project.version}</doctitle>
                     <windowtitle>Eclipse Collections - ${project.version}</windowtitle>
                     <show>public</show>
                     <detectJavaApiLink>true</detectJavaApiLink>
                     <links>
                         <link>https://junit.org/junit4/javadoc/latest</link>
-                        <link>https://docs.oracle.com/javase/8/docs/api/</link>
+                        <link>https://docs.oracle.com/en/java/javase/17/docs/api</link>
                     </links>
                     <destDir>${project.version}</destDir>
                     <doclint>html,syntax,accessibility</doclint>
                     <excludePackageNames>
-                        org.openjdk,org.eclipse.collections.impl.jmh,org.eclipse.collections.codegenerator,org.eclipse.collections.codegenerator.maven
+                        org.openjdk,org.eclipse.collections.impl.jmh,org.eclipse.collections.codegenerator.maven
```

Comment:
`excludePackageNames` doesn't seem to take as there is javadoc created for these excluded package names. I can revert this change, doesn't matter. I wanted to bring this to attention.

Reply:
I think this has to do with the javadoc plugin version. Also the delimiter might have changed: https://maven.apache.org/plugins/maven-javadoc-plugin/examples/exclude-package-names.html

Reply:
Delimiter has not changed. From the link you referenced: 

_If more than one package is to be excluded, each package name must be separated either with a colon (':'), a comma (',') or a semicolon (';')._